### DISCUSSION TITLE
Enable discussions please?

### DISCUSSION THREAD

**Discussion by dsyme** [dotnet/fsharp#10676] [General]
GitHub Discussions are now available.  Could one of the repo owners enable discussions in this repo please?  Under the main page of "settings"

Then actively move all issues that are labelled "discussion" to be discussions.

Thanks

**Comment by cartermp** [comment]
:)

**Reply by dsyme** [reply]
What do you think about moving all "Feature Request" issues too?

**Reply by cartermp** [reply]
I'd prefer not right now. Most are pretty straightforward.
